README.md

```
@@ -1,70 +1,96 @@
-
-<!--START_SECTION:activity-->
-<!--END_SECTION:activity-->
+---
 
+### 🧠 DevCard
 
-<!--
-**mgmgpyaesonewin/mgmgpyaesonewin** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.
+<p align="center">
+  <a href="https://app.daily.dev/pyaesonewin">
+    <img src="https://api.daily.dev/devcards/ecdc02169f8f47109380b815df096438.png?r=0ej" width="400" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)